azext_iot/common/embedded_cli.py

```diff
@@ -69,6 +69,9 @@ def success(self) -> bool:
         logger.debug("Operation error code: %s", self.error_code)
         return self.error_code == 0
 
+    def get_error(self) -> Exception:
```

Comment:
Typing of result should be `Union[None, Exception]` if either can be returned

Reply:
will change to `Optional[Exception]`

Reply:
Sounds good, but once the min python is 3.10 the most modern typing is `Exception | None`.